boards/components/src/rng.rs

```diff
@@ -88,3 +88,47 @@ impl<E: Entropy32<'static>> Component for RngComponent<E> {
         rng
     }
 }
+
+#[macro_export]
+macro_rules! rng_random_component_static {
+    ($R: ty $(,)?) => {{
+        let rng = kernel::static_buf!(capsules_core::rng::RngDriver<'static, $R>);
+
+        rng
+    };};
+}
+
+pub type RngRandomComponentType<R> = rng::RngDriver<'static, R>;
+
+pub struct RngRandomComponent<R: Rng<'static> + 'static> {
```

Comment:
What is the difference between `RngRandomComponent` and `RngComponent`? When would I choose one over the other? The commit messages seems to suggest this is a random something-or-other with no entropy which, I neither understand, nor do I see why it would ever be a very good idea.